### PR TITLE
Check, fix and add H2H weapons.

### PR DESCRIPTION
**Pull request by helixhamin**
Most effects are added, except the ones that are not yet applied in core.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/items/perfervid_sword.lua [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    dmg = adjustForTarget(target, dmg, xi.magic.ele.FIRE)
+    dmg = finalMagicNonSpellAdjustments(player, target, xi.magic.ele.FIRE, dmg)
+
+    local message = xi.msg.basic.ADD_EFFECT_DMG
+    if (dmg < 0) then
+        message = xi.msg.basic.ADD_EFFECT_HEAL
+    end
+
+    return xi.subEffect.FIRE_DAMAGE, message, dmg
+    end
```
Comment (zach2good) [inline scripts/globals/items/perfervid_sword.lua]:
```
Checking scripts/globals/items/perfervid_sword.lua 1 error
scripts/globals/items/perfervid_sword.lua:29:1: (E011) expected <eof> near 'end'
```

File: scripts/globals/items/pinion_cesti.lua [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onAdditionalEffect = function(player, target, damage)
+    local chance = 5
+
+    if (VanadielDayOfTheWeek() == xi.day.ICEDAY) then
+        chance = chance+6
+    end
```
Comment (zach2good) [inline scripts/globals/items/pinion_cesti.lua]: > [x] that I have read the Contributing Guide

For many many of these scripts:

https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations

**Review by zach2good — CHANGES_REQUESTED** [review]
I don't know how much of this can be taken without ruining @TeoTwawki's work on additional effects

File: scripts/globals/items/pinion_cesti.lua (hunk 2 above)
Comment (helixhamin) [reply scripts/globals/items/pinion_cesti.lua]: Sorry @zach2good , all of the additional effects items I made were copied from others, so wasn't changed other than values. So, I should remove the () after VanadielDayOfTheWeek? If so, I can work on fixing those.

Comment (zach2good) [reply scripts/globals/items/pinion_cesti.lua]: `VanadielDayOfTheWeek` is a function, so you need to call it with `VanadielDayOfTheWeek()`.

**Comment by helixhamin** [comment]
> I don't know how much of this can be taken without ruining @TeoTwawki's work on additional effects

@zach2good Removed the scripts completely. The additional effect needed items are still listed in the item_mods.sql as mod 431, but to prevent interference with Teo's redo of the additional effects, I am not commiting them, and all are commented out.

**Comment by TeoTwawki** [comment]
target branch need to change to https://github.com/LandSandBoat/server/tree/wip/effect-rework

once there you will see the edit conflicts

File: scripts/globals/items/pinion_cesti.lua (hunk 2 above)
Comment (TeoTwawki) [reply scripts/globals/items/pinion_cesti.lua]: one should understand the language before attempting to do these things

**Comment by TeoTwawki** [comment]
> to prevent interference with Teo's redo of the additional effects, I am not commiting them, and all are commented out.

That commenting out doesn't help. at all. as far as git is concerned those are still more changes to the same lines of the file.

Should I link all of the time I have asked that the add effect branch be targeted for add effect things?

**Comment by helixhamin** [comment]
> target branch need to change to https://github.com/LandSandBoat/server/tree/wip/effect-rework
> 
> once there you will see the edit conflicts

Sure, sorry, won't be able to until Monday. Sorry, might have to check with you since I'm not sure how to retarget, but I will ping you on discord for that.

**Comment by helixhamin** [comment]
Closed. I will retarget and try again in proper PR.

**Pull request closed without merging**